src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisabledDisplayNameProvider.cs

```diff
@@ -13,8 +13,10 @@ namespace Microsoft.PowerFx.Core
     [ThreadSafeImmutable]
     internal class DisabledDisplayNameProvider : DisplayNameProvider
     {
-        public static DisabledDisplayNameProvider Instance { get; } = new DisabledDisplayNameProvider();
-
+        public static DisabledDisplayNameProvider Instance { get; } = new DisabledDisplayNameProvider();
+
+        public override IEnumerable<KeyValuePair<DName, DName>> LogicalToDisplayPairs => throw new NotImplementedException();

```

Comment:
Wondering if this should throw, or just return an empty list. 
The Try* functions don't throw. 

Throwing may be safer - if we hit the exception, we can revisit.

Reply:
Empty List does sound better to me.